src/modules/roc_sndio/target_sox/roc_sndio/player.h

```diff
@@ -60,11 +58,23 @@ class Player : public core::Thread {
     //!  Should be called once before calling start().
     bool open(const char* name, const char* type = NULL);
 
+    //! Start reading samples.
+    //!
+    //! @b Parameters
+    //!  - @p input is used to read samples.
+    void start(pipeline::IReceiver& input);
+
     //! Stop thread.
     //! @remarks
     //!  Can be called from any thread.
     void stop();
 
+    //! Get a user defined or auto-detected sample rate of an output file or a device.
+    //!
+    //! @remarks
+    //! Output file or device should be opened.
```

Comment:
Two little notices:

* We can change `@remarks` to `@pre` (preconditions) here, it would fit a bit better.
* We usually indent `@remarks`, `@pre`, and similar sections, e.g.:

    ```
    //! @pre
    //!  Output file or device should be opened.
    ```

Reply:
Fixed